common/log/log_util.c

```diff
@@ -0,0 +1,35 @@
+#include <zephyr.h>
+#include <sys/printk.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include "log_util.h"
+
+#define log_name_to_num(x) #x,
+const char *const log_name[] = { log_name_to_num(DEBUG_I2C) log_name_to_num(DEBUG_PECI)
```

Comment:
Please reformat or run linter to make this list easier to read.

Reply:
Thank you for your reply, new commit has modified this format issue.